screens/TimerScreen.js

```diff
@@ -69,6 +71,22 @@ const TimerScreen = ({ navigation }) => {
     return;
   }
 
+  const handlePH = () => {
```

Comment:
generally just use the full spellings, we all got code editors for autocomplete so it just makes it more readable